tests/python_functional/src/syslog_ng_config/syslog_ng_config.py

```diff
@@ -81,8 +82,11 @@ def create_file_source(self, **options):
     def create_example_msg_generator_source(self, **options):
         return ExampleMsgGeneratorSource(**options)
 
+    def create_set_tag(self, tag, **options):
```

Comment:
I would rewrite this function to: `create_rewrite_set_tag()`

Reply:
Fixed.